presto-main/src/main/java/io/prestosql/server/security/oauth2/Status.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.server.security.oauth2;
+
+enum Status
+{
+    STARTED,
+    SUCCEEDED,
+    FAILED,
+    /**/;
```

Comment:
Do we expect to add new states? This trick of trailing comma is used to make it easy to add new enum values in the future. If not, write this as
```java
enum Status
{
    STARTED, SUCCEEDED, FAILED
}